cmd/infra/ibmcloud_powervs/create.go

```diff
@@ -139,42 +157,68 @@ func NewCreateCommand() *cobra.Command {
 	return cmd
 }
 
-func (options *CreateInfraOptions) Run(ctx context.Context) (err error) {
-
-	infra := &Infra{}
-	err = infra.setupInfra(options)
+func getEmptyStat() CreateStat {
+	return CreateStat{
+		Status:   "n/a",
+		Duration: "n/a",
+	}
+}
 
+func (options *CreateInfraOptions) Run(ctx context.Context) (err error) {
+	infra := &Infra{ID: options.InfraID,
+		Stats: InfraCreationStat{
+			Vpc:            getEmptyStat(),
+			VpcSubnet:      getEmptyStat(),
+			CloudInstance:  getEmptyStat(),
+			DhcpService:    getEmptyStat(),
+			CloudConnState: getEmptyStat(),
```

Comment:
Continuation of previous comments answer, if we are not able to generate this value for a specific resource, it would display as "n/a", that was my intention. 
Let me know your thoughts.

Reply:
in most of the case, it is good to omit the fields rather than displaying n/a, any non empty value is considered as a valid value.

Reply:
removed getEmptyStat().